app/controllers/shopping_list_items_controller.rb

```diff
@@ -24,19 +17,10 @@ def list_item_params
     params.require(:shopping_list_item).permit(
       :description,
       :quantity,
-      :notes,
-      :shopping_list_id
+      :notes
```

Comment:
Currently we don't need to pass the list ID in so I'm taking it out of the params so it doesn't confuse things.